### PR TITLE
added a question if the cso has an office

### PR DESCRIPTION
**Pull request by jpenanje**

### CODE REVIEW

**Review by nfebe — CHANGES_REQUESTED** [review]
I can't see the database migration, For these new fields.

**Review by nfebe — CHANGES_REQUESTED** [review]
Add all the 3 new columns in a single migration. (I think it's better that way).

File: database/migrations/2023_08_05_230419_add_fields_to_cso_registration_form.php [hunk 1]
```diff
@@ -22,7 +24,7 @@ public function up(): void
     public function down(): void
     {
         Schema::table('csos', function (Blueprint $table) {
-            //
+            Schema::dropIfExists('csos');
```
Comment (Fenn-CS) [inline database/migrations/2023_08_05_230419_add_fields_to_cso_registration_form.php:27]: The down method is the opposite of up! In this case, you added 3 rows, so in the down function you should delete/remove/undo the 3 columns.

Dropping the table would be fine if this migration was the first to introduce it.

**Review by nfebe — COMMENTED** [review]
1 more comment

**Comment by jpenanje** [comment]
> The number of staff selected when creating a cso on the frontend do not reflect in the cso dashboard edit page

The errors have been rectified.

'three-ten' => '3 - 10'  instead of 'three - ten' => '3 - 10' 
'eleven-fifty' => '11 - 50' 'eleven - fifty' => '11 - 50'

**Review by ExcelDurant — APPROVED** [review]

**Review by ExcelDurant — APPROVED** [review]

**Review by nfebe — APPROVED** [review]

**Pull request merged**